src/fx/History.js

```diff
@@ -0,0 +1,71 @@
+function historyPush(state, title, url) {
```

Comment:
Why were these extracted out into separate one liners?

Reply:
To be honest, I can't remember. I was wondering the same while implementing the ReplaceState effect. I'm partial to inlining them.